x-pack/plugins/fleet/public/components/agent_enrollment_flyout/agent_enrollment_flyout.test.tsx

```diff
@@ -88,10 +85,6 @@ describe('<AgentEnrollmentFlyout />', () => {
       agentPolicies: [{ id: 'fleet-server-policy' } as AgentPolicy],
       refreshAgentPolicies: jest.fn(),
     });
-
-    act(() => {
-      results = render();
-    });
```

Comment:
Also there's no need to render here, if we still invoke render again in each test suite